app/views/components/_option-select.html.erb

```diff
@@ -24,7 +24,7 @@
       }
     %>
   <% end %>
-  <%
+  <% # NOTE - could this be achieved with a <template> instead of putting it into a data attribute? Or by putting the filter_id into a data attribute to us?
```

Comment:
Let's remove this comment and add it into the backlog - https://trello.com/c/N16toILy/572-fix-minor-option-select-issues

```suggestion
  <% 
```